portfolio/src/main/webapp/index.html

```diff
@@ -78,9 +75,17 @@ <h3>Overview</h3>
             <br>
           </div>
           <div class="container">
+            <form action="/comments" method="GET">
```

Comment:
I don't think you need the form around the select dropdown, action and method doesn't do anything here, as they are handled in the refreshComments function